_data/meltano/extractors/tap-snapchat-ads/gthesheep.yml

```diff
@@ -53,6 +73,11 @@ settings_group_validation:
 - - refresh_token
   - client_secret
   - client_id
+  - start_date
+  - swipe_up_attribution_window
```

Comment:
@GtheSheep are these required settings? I saw that the `--about` [output](https://github.com/meltano/hub/pull/1052#issuecomment-1331344621) from the CI bot didnt require them.

Reply:
they aren't, is that what this `settings_group_validation` section is?

Reply:
@GtheSheep yeah thats a list of required settings. Check out the [plugin definition syntax docs](https://docs.meltano.com/reference/plugin-definition-syntax#settings_group_validation) for more detail

Reply:
Fair play, changed 👌